smart-table-module/js/Table.js

```diff
@@ -25,6 +25,9 @@
             scope.currentPage = 1;
             scope.holder = {isAllSelected: false};
 
+            // index of last row that was selected
+            this.prevouslySelectedIndex = null;
```

Comment:
I will change the code to set the variable also on "single" and "multiple" selectionModes in the click handler, so that this variable makes sense for all modes.